docs/make.jl

```diff
@@ -186,6 +186,13 @@ makedocs(;
         canonical = "https://docs.dftk.org/stable/",
         edit_link = "master",
         assets = ["assets/favicon.ico"],
+        mathengine = Documenter.MathJax(Dict(:TeX => Dict(
+            :Macros => Dict(
+                :ket => ["|#1\\rangle", 1],
+                :bra => ["\\langle#1|", 1],
+                :mel => ["\\langle#1|#2|#3\\rangle", 3],
```

Comment:
Nice idea. I usually define these guys (which automatically scale the bars and brakets to an appropriate size):
```tex
\newcommand{\bra}[1]{\left\langle#1\right|}
\newcommand{\ket}[1]{\left|#1\right\rangle}
\newcommand{\braket}[2]{\left\langle#1\middle|#2\right\rangle}
```
I don't use the `\mel` because this can be easily made using `\braket` plus an extra `\middle|`, so that gives some extra flexibility.